### ISSUE TITLE
drag-resize doesn't work when instantiated inside other polymer-element

### ISSUE THREAD

**Issue by gbisheimer** [manekinekko/drag-resize#2]
This is because the mousedown event is attached to `document` instead of `parentNode` in `domReady` function.

**Comment by manekinekko** [comment]
can you submit a PR please?

**Comment by gbisheimer** [comment]
Yes, as soon as I finish testing the changes... Also added a keyboard handler to move and scale things using keyboard.
